react-app/src/components/projects/ProjectDetail.tsx

```diff
@@ -398,6 +404,39 @@ const ProjectDetail = (props: IProjectDetail) => {
             )}
           </Box>
         </DataCard>
+        <DataCard
+          loading={isLoading}
+          title={notifications}
+          values={undefined}
+          id={notifications}
+          onEdit={() => setOpenNotificationDialog(true)}
+          disableEdit={isAuditor}
+          editButtonText="View Updated Notifications"
+        >
+          {!data?.parsedBody.Notifications?.length ? ( //TODO: Logic will depend on precense of agency responses
+            <Box display={'flex'} justifyContent={'center'}>
+              <Typography>No notifications were sent for this project.</Typography>
+            </Box>
+          ) : (
+            <ProjectNotificationsTable
+              rows={
+                data?.parsedBody.Notifications
+                  ? data?.parsedBody.Notifications.sort(
```

Comment:
Would probably make more sense to provide a default sort order to the GridColDef of this table component rather than sort here.